### PR TITLE
check travis is working

### PR DESCRIPTION
**Pull request by fain182**
I fear test are not really running on travis, they give no output and it's always green, like for example in #62 where locally the test fails with php minor then 5.6.

### CODE REVIEW

**Comment by fain182** [comment]
ok, confirmed that is broken :-)

**Comment by fain182** [comment]
Now it's working, the problem was that:
 - to run the test is used /bin/idephix.phar that is old and don't support the new style of idxfile

The solutions are:
 - use /bin/idx to run the tests that is always updated (this is what I did now)
 - update the /bin/idephix.phar once and hope that doesn't will happen again
 - other ?

**Comment by micheleorselli** [comment]
We should aim to stability hence updating the idephix version every build seems risky to me. I vote for upgrading idx once

**Comment by fain182** [comment]
I have another idea, what about dowloading the last stable idephix.phar during the build and use that?

**Comment by micheleorselli** [comment]
It depends on what "stable" means :-). Imo the build tool should be upgraded wisely and shouldn't be a moving part in the build script

**Comment by fain182** [comment]
Maybe we should use the latest release?
(for example now it would be the 0.0.1 that we know that works, and we will revert the idxfile to the old style until the next stable release)

**Comment by fain182** [comment]
Ok, now travis is working and is failing as expected.
For me we can merge this and open other pull request to fix all the other problems that travis shows.

File: .travis.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ php:
   - 5.5
 
 before_script:
-  - composer self-update
+  - composer install
```
Comment (micheleorselli) [inline .travis.yml]: is this composer install stil needed?

**Comment by fain182** [comment]
ok, now it should be everything right...

**Comment by micheleorselli** [comment]
So is the build supposed to fail? https://travis-ci.org/ideatosrl/Idephix/jobs/130896952

**Comment by fain182** [comment]
Exactly, because actually even master is failing, but travis is not showing it (because tests are not running).

And there are at least another couple of different issue that are still to solve:
 - there are some new parts that are not compatible with php 5.3 (because they use the new array syntax)
 - a bigger issue that i wrote in the comment: https://github.com/ideatosrl/Idephix/pull/62#issuecomment-212798359

And I think is better to fix one problem per pull request.

**Comment by micheleorselli** [comment]
ok, going to merge that

**Pull request merged**